TrafficCapture/captureKafkaOffloader/build.gradle

```diff
@@ -2,6 +2,7 @@
 plugins {
     id 'org.opensearch.migrations.java-library-conventions'
     id("io.freefair.lombok") version "8.0.1"
+    id 'org.owasp.dependencycheck' version '8.2.1'
```

Comment:
We should put this line in the top-level gradle file and have it apply to all projects rather than rely upon each project including boilerplate.

Reply:
Addressed in new commit. Removed the plugin from all "child" gradle files and added it to top-level.
Waiting for checks to pass then merging.